mergify_engine/web/api/queues.py

```diff
@@ -84,14 +84,14 @@ class SpeculativeCheckPullRequest:
 
 @pydantic.dataclasses.dataclass
 class QueueRule:
-    name: rules.QueueName = dataclasses.field(
-        metadata={"description": "The name of the queue rule"}
-    )
-
     config: rules.QueueConfig = dataclasses.field(
         metadata={"description": "The configuration of the queue rule"}
     )
 
+    name: rules.QueueName = dataclasses.field(
+        metadata={"description": "The name of the queue rule"}
+    )
```

Comment:
Why moving this?

Reply:
alphabetic order in class definition for maintainability

Reply:
I don't think it's a good idea: it just moves the code, break tools like `git blame` and it's not enforce by any automated check. So that's never going to stay like this forever. 😞